source/initial-dns-seedlist-discovery/initial-dns-seedlist-discovery.rst

```diff
@@ -32,6 +32,12 @@ domain to return a list of host names. Supporting this feature would assist
 our users by decreasing maintenance load, primarily by removing the need to
 maintain seed lists at an application level.
 
+This specification builds on the `Connection String`_ specification. It adds a
+new protocol scheme and modifies how the `Host Information`_ is interpreted.
+
+.. _`Connection String`: ../connection-string/connection-string-spec.rst
+.. _`Host Information`: ../connection-string/connection-string-spec.rst#host-information
```

Comment:
Yeah, I guess we have to accept that we're never going to really publish these aside from uploading them to GitHub, so links within GitHub are fine.

Reply:
Yeah I kind of thought these belonged in a "Terminology" section, but I think this is fine as well.